src/components/routerView.browser.spec.ts

```diff
@@ -188,6 +189,10 @@ test('Renders custom genericRejection component when the initialUrl does not mat
     },
   })
 
+  if (!isRouteWithComponent(router.route.matched)) {
+    throw 'Matched route does not have a single component'
+  }
```

Comment:
I'm not 100% sold on the types here. See `routeProps.ts` for why this is necessary